### ISSUE TITLE
Touchpad and Touch Screen Issue in KDE

### ISSUE THREAD

**Issue by pasmon79** [rickysarraf/laptop-mode-tools#180]
Sometimes on restart my touchscreen and touch-pad fails to load. 

Can laptop-mode-tools be contributing to this. 

Kindly indicate the settings that affect these two items on my linux plasma.

**Comment by rickysarraf** [comment]
Possibly. The `runtime-pm` module can hinder such devices. The best way to confirm that is by disabling `laptop-mode-tools` in `/etc/laptop-mode/laptop-mode.conf` by setting variable `ENABLE_LAPTOP_MODE_TOOLS=0` and the reboot.

Please do share the results and then we can look further into the cause.

**Comment by pasmon79** [comment]
I already unchecked all the options in Laptop-mode, uninstalled it and removed /etc/laptop-mode completely. 
Yet some signatures remained, one of which disabled my wifi completely. I had to edit the default-wifi-powersave-on to take care of it.

I am absolutely sure that laptop-mode has edited some essential system file "not part of laptop-mode" conflicting with touch drivers.

**Comment by rickysarraf** [comment]
> I already unchecked all the options in Laptop-mode, uninstalled it and removed /etc/laptop-mode completely. Yet some signatures remained, one of which disabled my wifi completely. I had to edit the default-wifi-powersave-on to take care of it.
> 
> I am absolutely sure that laptop-mode has edited some essential system file "not part of laptop-mode" conflicting with touch drivers.

Then you have a different problem. Possibly directly related to your device/driver. Nothing that `laptop-mode` does is persisted across reboots.

**Comment by pasmon79** [comment]
Kindly guide me if laptop-mode tools has written over the files involving touch devices or not.

Thanks for your kind guidance.

**Comment by rickysarraf** [comment]
> Kindly guide me if laptop-mode tools has written over the files involving touch devices or not.
> 

No. You must be having a different problem altogether. There's nothing here, from what you describe, that could now be caused by `laptop-mode-tools`. Once it is disabled, there's nothing it does.
